### DISCUSSION TITLE
I cant figure out what I'm doing wrong.

### DISCUSSION THREAD

**Discussion by JollyGreen21** [PromtEngineer/localGPT#305] [Q&A] [answered]
Can someone that has this working on a windows computer please post their "pip freeze" and the "nvcc --version" from cmd line.

I would really like to try this out, but I have been mucking around for days and I don't feel any closer to making this work.

**Comment by JollyGreen21** [comment, marked as answer]
Looks like bitsandbytes does not work on windows... would be nice if it said that at the TOP of the readme not 3/4 down.

**Comment by pseudotensor** [comment]
It works fine.  You can try h2oGPT on windows.  I just ran through instructions a couple days ago and give examples of 4-bit and 8-bit, including timing and memory usage.

https://github.com/h2oai/h2ogpt/blob/main/docs/README_WINDOWS.md
